echoregions/utils/api.py

```diff
@@ -3,29 +3,39 @@
 
 import numpy as np
 import xarray as xr
-from xarray import DataArray, Dataset
+from xarray import Dataset
 
 
-def convert_mask_2d_to_3d(mask_2d: DataArray) -> Union[Dataset, None]:
+def convert_mask_2d_to_3d(mask_2d_ds: Dataset) -> Union[Dataset, None]:
     """
     Convert 2D multi-labeled mask data into its 3D one-hot encoded form.
 
     Parameters
     ----------
-    mask_2d: DataArray
-        A DataArray with the data_var masked by a specified region. Individual data
-        points will be in the form of integers, demarking region_id of masked regions,
-        and nan values, demarking non-masked areas.
+    mask_2d_ds : Dataset
+        A dataset with the following:
+            DataArray with the data_var masked by a specified region. Individual data
+            points will be in the form of integers, demarking region_id of masked regions,
+            and nan values, demarking non-masked areas.
+            DataArray with mask labels corresponding to region_id values.
 
     Returns
     -------
-    mask_3d : Data Array
-        A 3D mask where each layer of the mask will contain a 1s/0s mask for each
-        unique label in the 2D mask. The layers will be labeled via region_id values
-        extracted from 2d values.
+    mask_3d_ds : Dataset
+        A dataset with the following:
+            A DataArray 3D mask where each layer of the mask will contain a 1s/0s mask for
+            each unique label in the 2D mask. The layers will be labeled via region_id
+            values extracted from 2d values.
+            DataArray with mask labels corresponding to region_id values.
     """
```

Comment:
```suggestion
    mask_3d_ds : Dataset
        A dataset with the following:
            A DataArray 3D mask where each layer of the mask will contain a 1s/0s mask for
            each unique label in the 2D mask. The layers will be labeled via region_id
            values extracted from 2d values.
            DataArray with mask labels corresponding to region_id values.
    
    Notes
    ------
    Note the output 3D mask Dataset does not contain the data variable `mask_labels`
    since this information is not available in the input 2D mask.
    """
```

Reply:
In fact `mask_labels` is retained in the 3d to 2d output.